assembl/static2/js/app/searchv1.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import Thunk from 'redux-thunk';
+import { loadTranslations, setLocale, syncTranslationWithStore } from 'react-redux-i18n';
+import RootReducer from './reducers/rootReducer';
+import GlobalFunctions from './utils/globalFunctions';
+import Translations from './utils/translations';
+import { canUseExpertInterface } from './utils/permissions';
+import { fetchPermissionsForUser } from './services/userService';
+
+import { SearchComponent } from './components/search.jsx?v=1'; // eslint-disable-line
+import '../../css/views/searchv1.scss';
+/*
+searchv1.js is included in assembl/templates/index.jinja2
+(you need to change the url for production)
+The search div is included in the page from assembl/templates/views/navBar.tmpl
+and the Search react component is rendered via the onDomRefresh event in
+assembl/static/js/app/views/navBar.js
+*/
+
+
+const myCreateStore = () => {
+  const store = createStore(RootReducer, applyMiddleware(Thunk));
+  const assemblLocale = window.assembl_locale.split('_')[0];
+  const userLocale = GlobalFunctions.getLocale(assemblLocale);
+  syncTranslationWithStore(store);
+  store.dispatch(loadTranslations(Translations));
+  store.dispatch(setLocale(userLocale));
+  return store;
```

Comment:
I don't understand this commit https://github.com/assembl/assembl/pull/79/commits/e7d617d298dde0a1c72535d7ca13cab5ba633f0c , maybe that's just because I'm not familiar with react-redux-i18n.